tools/machine-learning/mujoco/packages/nao_env/src/nao_env/nao_standing.py

```diff
@@ -0,0 +1,133 @@
+from pathlib import Path
+from typing import ClassVar
+
+import numpy as np
+import rewards
+from gymnasium import utils
+from gymnasium.envs.mujoco.mujoco_env import MujocoEnv
+from gymnasium.spaces import Box
+from nao_interface.nao_interface import Nao
+from nao_interface.poses import PENALIZED_POSE
+from throwing import ThrowableObject
+
+DEFAULT_CAMERA_CONFIG = {
+    "trackbodyid": 1,
+    "distance": 4.0,
+    "lookat": np.array((0.0, 0.0, 0.8925)),
+    "elevation": -20.0,
+}
+
+OFFSET_QPOS = np.array(
+    [
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.09,
+        -0.06,
+        0.01,
+        -0.002,
+        0.0,
+        0.09,
+        -0.06,
+        0.01,
+        0.002,
+        1.57,
+        0.1,
+        -1.57,
+        0.0,
+        0.0,
+        1.57,
+        -0.1,
+        1.57,
+        0.0,
+        0.0,
+    ]
+)
+
+HEAD_SET_HEIGHT = 5.11757778e-01
+
+
+class NaoStanding(MujocoEnv, utils.EzPickle):
+    metadata: ClassVar = {
+        "render_modes": [
+            "human",
+            "rgb_array",
+            "depth_array",
+        ],
+        "render_fps": 83,
+    }
+
+    def __init__(self, throw_tomatos: bool, **kwargs) -> None:
+        observation_space = Box(
+            low=-np.inf,
+            high=np.inf,
+            shape=(37,),
+            dtype=np.float64,
+        )
+
+        MujocoEnv.__init__(
+            self,
+            str(Path.cwd().joinpath("model", "scene.xml")),
+            frame_skip=4,
+            observation_space=observation_space,
+            default_camera_config=DEFAULT_CAMERA_CONFIG,
+            **kwargs,
+        )
+        self.throw_tomatos = throw_tomatos
+        self.projectile = ThrowableObject(
+            self.model, self.data, "floor", "tomato"
+        )
+        self.current_step = 0
+        utils.EzPickle.__init__(self, **kwargs)
+
+    def _get_obs(self) -> np.ndarray:
```

Comment:
Is this an override? then mark it as
```suggestion
    @override
    def _get_obs(self) -> np.ndarray:
```